packages/framework/src/Resources/views/Admin/Grid/Grid.html.twig

```diff
@@ -102,25 +102,37 @@
         {% endif %}
         <a class="in-sort-arrows" href="{{ gridView.url({'order': orderBy}) }}">
             <span class="in-sort-arrows__link {% if grid.orderSourceColumnName == column.id %} active{% endif %}">{{ column.title }}</span>
-            {% if column.help is not null %}
-                <span title="{{ column.help }}"
-                      class="js-tooltip cursor-help in-icon in-icon--in-text in-icon--info"
-                      data-toggle="tooltip"
-                      data-placement="bottom"
-                >
-                    {{ ux_icon('info') }}
-                </span>
-            {% endif %}
             {% if grid.orderSourceColumnName == column.id %}
                 {% if grid.orderDirection == 'desc' %}
                     <span class="in-sort-arrows__item in-sort-arrows__item--down"></span>
                 {% else %}
                     <span class="in-sort-arrows__item in-sort-arrows__item--up"></span>
                 {% endif %}
             {% endif %}
+            {% if column.help is not null %}
+                <span title="{{ column.help }}"
+                    class="js-tooltip cursor-help in-icon in-icon--in-text in-icon--info"
+                    data-toggle="tooltip"
+                    data-placement="bottom"
+                    {% if grid.orderSourceColumnName == column.id %}
+                        style="margin-left: 20px;"
+                    {% endif %}
+                >
+                    {{ ux_icon('info') }}
+                </span>
+            {% endif %}
```

Comment:
this could be extracted into a macro to avoid the duplication :innocent:

Reply:
Yes, but they are not the same and after implementing the new administration it will be changed. I will not change it now :)